flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/util/StormFileSpout.java

```diff
@@ -45,9 +49,16 @@ public StormFileSpout(final String path) {
 	public void open(final Map conf, final TopologyContext context, final SpoutOutputCollector collector) {
 		super.open(conf, context, collector);
 		try {
+			/* if inputFile path has not set, get it from storm configuration  */
+			if (this.path == null && conf != null && conf.containsKey(new String("textpath"))) {
```

Comment:
I would not check for `this.path == null`. If config is given, it should override constructor path.